shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/main/java/org/apache/shardingsphere/proxy/frontend/command/CommandExecutorTask.java

```diff
@@ -66,8 +69,11 @@
     @Override
     public void run() {
         boolean isNeedFlush = false;
+        boolean sqlShowEnabled = isSQLShowEnabled();
```

Comment:
```suggestion
        boolean sqlShowEnabled = ProxyContext.getInstance().getContextManager().getMetaDataContexts().getProps().getValue(ConfigurationPropertyKey.SQL_SHOW);
```
Could we consider doing inline manually?

Reply:
Thank you @TeslaCN , updated.